suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
@vuntz : Running rabbitmqctl from rabbitmq user is better idea than doing this from root account. It least `/usr/sbin/rabbitmqctl` is calling `/usr/lib64/rabbitmq/bin/rabbitmqctl` from rabbitmq user.